addons/link_tracker/tests/test_merge_utm_campaign.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tools import mute_logger
+from odoo.tests.common import TransactionCase
```

Comment:
We usually use `SavepointCase` when it's possible, as it's faster than `TransactionCase` and gives almost the same mechanisms.